### PR TITLE
Feat/leaflet integration

### LINKED ISSUE #6: ## Add a Map
As a user,
I want to see a map,
so that I know what is around me

- [x] implement a map 
- [x] link the map to the menu

### PR DESCRIPTION
**Pull request by Marc-Klein**
## Motivation

- The app should show a Basic Map
- The map should be reachable from the Menu
- The map should be reloadable  

## Issues closed
close #6

### CODE REVIEW

File: src/atoms/initialize-map/index.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
+import styled from "@emotion/styled";
+
+const StyledMapContainer = styled(MapContainer)`
+	width: 100vw;
+	height: 100vh;
+	margin: 0;
+	padding: 0;
+`;
+
```
Comment (doemser) [inline src/atoms/initialize-map/index.jsx:13]: you could put this styled component next to the index.jsx as styled.js

File: src/pages/index.jsx [hunk 2]
```diff
@@ -7,8 +7,18 @@ const Page = () => {
...
+					name="viewport"
+					content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+				/>
+				{/*maybe not nescecari*/}
+				{/*<link*/}
+				{/*	rel="stylesheet"*/}
+				{/*	href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"*/}
+				{/*	integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="*/}
+				{/*	crossOrigin=""*/}
+				{/*/>*/}
```
Comment (doemser) [inline src/pages/index.jsx:21]: consider using scratch-files for code that is not running

**Review by doemser — APPROVED** [review]
Yo,  looks great! I left you some comments you could consider.

File: src/pages/index.jsx (hunk 2 above)
Comment (Marc-Klein) [reply src/pages/index.jsx:21]: Thanks for the suggestion, but I deleted it anyway after the push.

Comment (doemser) [reply src/pages/index.jsx:21]: than you could commit it to the same branch and PR and  give your code to review again

File: src/atoms/initialize-map/index.jsx (hunk 1 above)
Comment (Marc-Klein) [reply src/atoms/initialize-map/index.jsx:13]: Done, thanks for the tip. ✅

**Review by doemser — APPROVED** [review]
nice :) lgtm

**Pull request merged**